### PR TITLE
Add active shuffling cache

### PR DESCRIPTION
**Pull request by AndrewBoyarsky**

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java [hunk 1]
```diff
@@ -59,13 +59,25 @@
...
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.CDI;
+import javax.inject.Singleton;
 
 public final class Shuffling {
+    /**
+     * Cache, which contains all active shufflings required for processing on each block push
+     * Purpose: getActiveShufflings db call require at least 50-100ms, with cache we can shorten time to 10-20ms
+     */
+    private static final Map<Long, Shuffling> activeShufflings = new HashMap<>();
```
Comment (yuriy-larin-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java]: I propose name it as 'activeShufflingsCache' implicitly.

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java [hunk 2]
```diff
@@ -450,6 +477,19 @@ private void setStage(Stage stage, long assigneeAccountId, short blocksRemaining
                 Long.toUnsignedString(id), this.stage, Long.toUnsignedString(this.assigneeAccountId), this.blocksRemaining);
     }
 
+    private static void insert(Shuffling shuffling) {
```
Comment (yuriy-larin-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java]: insertToTableCache(...)  ??

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java (hunk 1 above)
Comment (AndrewBoyarsky) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java]: Fixed

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java (hunk 2 above)
Comment (AndrewBoyarsky) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java]: Will be renamed during refactoring

**Pull request merged**